Error "Module '"./service.type"' has no exported member 'TranslateService'." when trying to bootstrap a host project

Hi.

Your project is very interesting and I am trying  to integrate it into a Module Federation Proof of Concept I am trying to build.  Unfortunately, I have a compilation when trying to compile my host project.

Using Angular 14.2  and @angular-architects/module-federation 14.2.  I applied the bootstrapping like mentioned in your Getting Started section.

But when I try to compile my project, I get the following error:

```
Error: node_modules/ng-dynamic-mf/lib/translations/loaded-translations.service.d.ts:1:10 - error TS2305: Module '"./service.type"' has no exported member 'TranslateService'.

1 import { TranslateService } from './service.type';
```

I added ngx-translate even though I am not using any translation features, but I still get the same error.

--- a/package.json
+++ b/package.json
@@ -47,7 +47,6 @@
     "@angular/platform-browser": "^14.2.0",
     "@angular/platform-browser-dynamic": "^14.2.0",
     "@angular/router": "^14.2.0",
-    "@ngx-translate/core": "^14.0.0",
     "rxjs": "~7.5.0",
     "tslib": "^2.3.0",
     "zone.js": "~0.11.4"

--- a/projects/ng-dynamic-mf/package.json
+++ b/projects/ng-dynamic-mf/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ng-dynamic-mf",
-  "version": "1.7.1",
+  "version": "1.8.1",
   "license": "MIT",
   "author": {
     "name": "Janik Schumacher (LoaderB0T)",

--- a/projects/ng-dynamic-mf/src/lib/translations/service.type.ts
+++ b/projects/ng-dynamic-mf/src/lib/translations/service.type.ts
@@ -1,4 +1,3 @@
-// @internal
-import { TranslateService as realType } from '@ngx-translate/core';
-// @internal
-export type TranslateService = realType;
+export interface TranslateService {
+  setTranslation(lang: string, translations: Object, shouldMerge?: boolean | undefined): void;
+}

